src/cmd/services/m3query/config/config.go

```diff
@@ -112,6 +113,9 @@ type Configuration struct {
 	// WriteWorkerPool is the worker pool policy for write requests.
 	WriteWorkerPool xconfig.WorkerPoolPolicy `yaml:"writeWorkerPoolPolicy"`
 
+	// WriteForwarding is the write forwarding options.
+	WriteForwarding remote.PromWriteHandlerForwardingOptions `yaml:"writeForwarding"`
```

Comment:
super nit: Would be nice to either have this be:

```
writeForwarding:
    prom:
        ....
```

or

```
prom:
   writeForwarding:
       ....
```

Basically this feature seems a little niche to be a top level key imo

Reply:
Yeah I debated it, not opposed to refactoring to the first suggestion.  Updating.